android/app/src/main/java/com/example/myos/ui/ui/product/ProductViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.myos.ui.ui.product
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class ProductViewModel : ViewModel() {
+
+    private val _text = MutableLiveData<String>().apply {
+        value = "Tela de Produtos"
```

Comment:
I18n?

Reply:
This String is for testing, it will not exist, it was only for viewing on the screen